platform/MCU/NRF51/SPARK_Firmware_Driver/inc/rtc_timer.h

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Is this class being used at all? Or is everything called from hal/src/bluz/apptimer_hal.c and that is just calling directly into the nrf51 driver?